apps/reviews-api/review/data/repository/review_repo_impl.go

```diff
@@ -107,6 +126,14 @@ func (r *reviewRepositoryImpl) SubmitReview(ctx context.Context, review *db.Revi
 }
 
 func (r *reviewRepositoryImpl) EditReview(ctx context.Context, review *db.ReviewModel, courseCode, userID string) (*db.ReviewModel, error) {
+	if err := isTermValid(review); err != nil {
+		return nil, err
+	}
+
+	if err := isSectionValid(review); err != nil {
+		return nil, err
+	}
```

Comment:
Where is the `GetBoolEnv` and feature flag for this XD

Reply:
OH SHOOT GOOD CATCH